echopype/echodata/echodata.py

```diff
@@ -59,12 +61,19 @@ def __init__(
         self.xml_path: Optional["PathHint"] = xml_path
         self.sonar_model: Optional["SonarModelsHint"] = sonar_model
         self.converted_raw_path: Optional["PathHint"] = None
+        self._tree: DataTree = None
 
         self.__setup_groups()
-        self.__read_converted(converted_raw_path)
+        # self.__read_converted(converted_raw_path)
```

Comment:
@b-reyes Note that I commented this out. This uses the old `_load_file` functionality.